apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -519,7 +527,8 @@ export class ChangeAction extends BaseRushAction {
    * Will determine a user's email by first detecting it from their Git config,
    * or will ask for it if it is not found or the Git config is wrong.
    */
-  private async _detectOrAskForEmail(promptModule: inquirer.PromptModule): Promise<string> {
+  // eslint-disable-next-line
+  private async _detectOrAskForEmail(promptModule): Promise<string> {
```

Comment:
I'm uncomfortable with dropping the typesafety here.

Can you import inquirer via `import type` and just use that for typing information?

Reply:
Rush is on TS 3.5, import type requires at least 3.8.

Reply:
As I explained with `IYarnLockfileParseResult`, just use a regular `import` with a TODO comment saying to convert to  "import type" later.  *Don't remove our type annotations.*